contributors.json

```diff
@@ -0,0 +1,85 @@
+[
+  {
+    "avatarUrl": "https://avatars.githubusercontent.com/u/42497995?v=4",
+    "name": "@androz2091",
+    "description": "Back-end API developer",
```

Comment:
Don't you want to say you're the one in charge / founder / whatever ? @Androz2091

Reply:
Not really. People can use contact@dumpus.app to contact us